pages/index.html

```diff
@@ -4,9 +4,18 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <title>Welcome!</title>
 </head>
 <body>
-  
+  <header class="header"><h1 class="header-text">Welcome to Brooke's Portfolio Website!</h1></header> 
+    <nav class="nav">
+        <li><a href="file:///Users/brookevlahos/Developer/projects/personal-portfolio-site/pages/index.html">Home</a></li>
+        <li><a href="file:///Users/brookevlahos/Developer/projects/personal-portfolio-site/pages/about.html">About</a></li>
+        <li><a href="file:///Users/brookevlahos/Developer/projects/personal-portfolio-site/pages/portfolio.html">Portfolio</a></li>
```

Comment:
The `li` element should go inside a `ul` or `ol` tag.

Also you've given **absolute** paths to the files **on your computer**.  The path is *different* on my computer.  To fix this use relative paths. 

```suggestion
      <ul>
        <li><a href="index.html">Home</a></li>
        <li><a href="about.html">About</a></li>
        <li><a href="portfolio.html">Portfolio</a></li>
      </ul>
```